[release/v1.4] Apply linting rules to mimir.* component docs (#2680)
grafana/alloy: docs/sources/reference/components/mimir/mimir.rules.kubernetes.md

`prometheus_http_prefix` is ignored if `use_legacy_routes` is set to `true`.

`external_labels` overrides label values if labels with the same names already exist inside the rule.

## Blocks

> The following blocks are supported inside the definition of
> `mimir.rules.kubernetes`:

Author: ```suggestion
You can use the following blocks with `mimir.rules.kubernetes`:
```
